hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -3032,6 +3049,32 @@ func getPullSecretName(ctx context.Context, crclient client.Client, hostedCluste
 	return pullSecret.Name, nil
 }
 
+func (r *NodePoolReconciler) getAdditionalTrustBundle(ctx context.Context, hostedCluster *hyperv1.HostedCluster) (string, error) {
+	additionalTrustBundle := &corev1.ConfigMap{}
+	if err := r.Client.Get(ctx, client.ObjectKey{Namespace: hostedCluster.Namespace, Name: hostedCluster.Spec.AdditionalTrustBundle.Name}, additionalTrustBundle); err != nil {
+		return "", fmt.Errorf("cannot get additionalTrustBundle %s/%s: %w", hostedCluster.Namespace, hostedCluster.Spec.AdditionalTrustBundle.Name, err)
```

Comment:
Should there be an additional check here to not return an error if it's just not found? This field is optional in the API.

Reply:
The only time this function gets called is when the additionalTrustBundle is specified in the HC.